server/apiCalls/transformers/analyses.test.js

```diff
@@ -13,10 +13,16 @@ const offering_SF1625_202121 = {
   courseCode: 'SF1625',
   offeringId: '1',
   period: 'P1',
+  courseRoundApplications: [
+    {
+      course_round_application_code: '1',
+    },
+  ],
 }
 const analysis_SF1625_202121_base = (courseCode = 'SF1625') => ({
   courseCode,
-  roundIdList: '1',
+  roundList: '1',
```

Comment:
`roundList ` has to be removed to be sure tests working properly

Reply:
Removed